astro/src/content/docs/apis/_login-response-codes-authenticated.mdx

```diff
@@ -2,22 +2,24 @@ import JSON from 'src/components/JSON.astro';
-| 213  | The user's registration has not yet been verified. The response will contain the User object that was authenticated. If the <InlineField>emailConfiguration.verificationStrategy</InlineField> has been set to `FormField`, the response will contain the <InlineField>registrationVerificationId</InlineField> that was generated for the user. <br /><br/> Prior to version `1.27.0`, this status code was not returned, and you will see a `200` instead.                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
-| 242  | The user was authenticated successfully. The user has two factor authentication enabled. Since version `1.42.0`, this status code is also returned when two factor authentication is required. The response will contain the `twoFactorId` to be used on the [Complete Two Factor Authentication](/docs/apis/login#complete-multi-factor-authentication) API.<br/><br/><JSON title="Example Response JSON" src="login/login-two-factor-response.json" />                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| 212  | The user's email address (if an email address or username was used to authenticate) or phone number has not yet been verified. The response will contain the User object that was authenticated. If the tenant verification strategy has been set to `FormField`, the response will contain the verification Id that was generated for the user.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
It might be helpful to begin the 212 and 213 response descriptions with `The user was authenticated, however...` so that it's clear that the authentication succeeded. We say that for all of the other 2xx responses.

Reply:
Fixed